lab5/lab5.rst

```diff
@@ -68,7 +68,7 @@ I don't know about tsch, etc.
 
     for i in 10 11 12 13 14
     do
-        ./cache_simulator ../../lab5/long_trace.txt $i 1 | grep Misses
+        ./cache_simulator ../../lab5/long_trace.txt $i 2 4 | grep Misses
```

Comment:
Can you update with 1 and 1 instead of 2 and 4? They are just random numbers either way...

Reply:
Done!